### PR TITLE
[DOC] update the supported OS in download page [skip ci]

### PR DESCRIPTION
**Pull request by nvliyuan**
we drop support centos7 now, updated the download doc.

### CODE REVIEW

**Review by sameerz — APPROVED** [review]

**Comment by sameerz** [comment]
build

File: docs/download.md [hunk 1]
```diff
@@ -27,7 +27,8 @@ The plugin is tested on the following architectures:
 
 ### Software Requirements:
 
-	OS: Ubuntu 20.04, Ubuntu 22.04, CentOS 7, or Rocky Linux 8
+    OS: Spark RAPIDS is compatible with any Linux distribution with glibc >= 2.28 (Please check ldd --version output).
+    Tested on Ubuntu 20.04, Ubuntu 22.04, Rocky Linux 8 and Rocky Linux 9
```
Comment (sameerz) [inline docs/download.md:31]: @revans2 suggested we should include the year that the glibc version was published, so users understand the breadth of linux distributions that should be supported.  

```suggestion
    OS: Spark RAPIDS is compatible with any Linux distribution with glibc >= 2.28 (Please check ldd --version output).  glibc 2.28 was released August 1, 2018. 
    Tested on Ubuntu 20.04, Ubuntu 22.04, Rocky Linux 8 and Rocky Linux 9
```

Ref: https://sourceware.org/glibc/wiki/Release/2.28

**Review by sameerz — COMMENTED** [review]
@revans2 suggested we should include the year that the glibc version was published, so users understand the breadth of linux distributions that should be supported.

**Review by sameerz — APPROVED** [review]

**Comment by nvliyuan** [comment]
build

**Comment by NvTimLiu** [comment]
build

**Pull request merged**